homeassistant/components/air_quality/waqi.py

```diff
@@ -11,37 +11,26 @@
 import aiohttp
 import voluptuous as vol
 
+from homeassistant.components.air_quality import (
+    PLATFORM_SCHEMA, AirQualityEntity, PROP_TO_ATTR)
 from homeassistant.exceptions import PlatformNotReady
 import homeassistant.helpers.config_validation as cv
-from homeassistant.const import (
-    ATTR_ATTRIBUTION, ATTR_TIME, ATTR_TEMPERATURE, CONF_TOKEN)
+from homeassistant.const import (ATTR_TIME, ATTR_TEMPERATURE, CONF_TOKEN)
 from homeassistant.helpers.aiohttp_client import async_get_clientsession
-from homeassistant.helpers.config_validation import PLATFORM_SCHEMA
-from homeassistant.helpers.entity import Entity
 
 REQUIREMENTS = ['waqiasync==1.0.0']
 
 _LOGGER = logging.getLogger(__name__)
 
 ATTR_DOMINENTPOL = 'dominentpol'
 ATTR_HUMIDITY = 'humidity'
-ATTR_NITROGEN_DIOXIDE = 'nitrogen_dioxide'
-ATTR_OZONE = 'ozone'
-ATTR_PM10 = 'pm_10'
-ATTR_PM2_5 = 'pm_2_5'
 ATTR_PRESSURE = 'pressure'
-ATTR_SULFUR_DIOXIDE = 'sulfur_dioxide'
-
-KEY_TO_ATTR = {
-    'pm25': ATTR_PM2_5,
-    'pm10': ATTR_PM10,
-    'h': ATTR_HUMIDITY,
-    'p': ATTR_PRESSURE,
-    't': ATTR_TEMPERATURE,
-    'o3': ATTR_OZONE,
-    'no2': ATTR_NITROGEN_DIOXIDE,
-    'so2': ATTR_SULFUR_DIOXIDE,
-}
+
+PROP_TO_ATTR.update({'temperature': ATTR_TEMPERATURE,
```

Comment:
Temperature, humidity and pressure are better implemented as sensors.

https://developers.home-assistant.io/docs/en/entity_sensor.html#available-device-classes

Reply:
It's in progress. Strangely enough it works except for the configuration.

if I have:

```python
CONFIG_SCHEMA = vol.Schema({
    DOMAIN: vol.Schema({
        vol.Optional(CONF_STATIONS): cv.ensure_list,
        vol.Required(CONF_TOKEN): cv.string,
        vol.Required(CONF_LOCATIONS): cv.ensure_list
    })
})
```

In the __init.py__ of the component, it always complains at launch:

```
2019-01-26 04:00:58 ERROR (MainThread) [homeassistant.config] Invalid config for [waqi]: [homeassistant] is an invalid option for [waqi]. Check: waqi->homeassistant. (See /Users/reefab/.homeassistant/configuration.yaml, line 170). Please check the docs at https://home-assistant.io/components/waqi/
2019-01-26 04:00:58 ERROR (MainThread) [homeassistant.setup] Setup failed for waqi: Invalid config.
```

With the config being: 

```yaml
waqi:
  token: TOKEN
  locations:
    - paris
  stations:
    - Paris Centre, Paris
```

Strangely enough, if I comment out the schema code, it works perfectly fine, the component see the correct config. I'm very probably missing out something obvious.

I'm stumped, but the rest of the code should be ready for review.